pkg/config/config.go

```diff
@@ -33,10 +33,15 @@ type Runtime struct {
 type ContainerdConfig struct {
 	// Snapshotter is the snapshotter used by containerd.
 	Snapshotter string `toml:"snapshotter" json:"snapshotter"`
-	// DefaultRuntime is the runtime to use in containerd.
+	// DefaultRuntime is the default runtime to use in containerd.
```

Comment:
The documentation for these fields has to be cut and pasted to here: https://github.com/containerd/cri/blame/master/docs/config.md#L52

Reply:
Thanks! I missed that.

Reply:
Done. Add a clarification about the example to the runtimes entry.